backend/src/schema/resolvers/rooms.ts

```diff
@@ -0,0 +1,56 @@
+import { v4 as uuid } from 'uuid'
+import { neo4jgraphql } from 'neo4j-graphql-js'
+import Resolver from './helpers/Resolver'
+
+export default {
+  Query: {
+    Room:  async (object, params, context, resolveInfo) => {
+      if (!params.filter) params.filter = {}
+      params.filter.users_some = {
+        id: context.user.id,
+      }
```

Comment:
This seems very hacky, but I assume thats how its suppose to be used?

You can hack this by providing the following filter:

```
_RoomFilter {
users_some: 'anything'
OR: _RoomFilter {
    users_some: 'my target user id'
  }
}
```
Remove the circular filters or the OR filter on `_RoomFilter`.

If you want to do it properly replace all `user_some` objects with the one defined in the code. Therfore you must walk the filter recursively. maybe loadash can provide an useful function